docs/index.md

```diff
@@ -70,6 +70,22 @@ Use Office Scripts in Excel on the web to automate your common tasks. Explore th
 <h2>Other resources</h2>
 <p>Use the following resources to learn about the APIs that Office Scripts use to interact with workbooks in Excel on the web, ask questions about Office Scripts in Excel on the web, or request features for Office Scripts in Excel on the web.</p>
 <ul class="panelContent cardsF cols cols4" style="display:flex!important;">
+    <li>
```

Comment:
For improved layout of tiles in the "Other resources" section, suggest changing `cols4` to `cols3` in the `<ul>` tag above (i.e., doing this will render the 6 tiles as 2 rows of 3 items-- instead of 1 row of 4 items + 1 row of 2 items)